src/Host/Broker/Impl/Sessions/SessionManager.cs

```diff
@@ -68,7 +69,8 @@ public class SessionManager {
                 password,
                 profilePath,
                 _loggingOptions.LogHostOutput ? _hostOutputLogger : null,
-                _loggingOptions.LogPackets ? _messageLogger : null);
+                _loggingOptions.LogPackets ? _messageLogger : null,
+                _loggingOptions.LogPackets ? LogVerbosity.Traffic : LogVerbosity.Minimal);
```

Comment:
Do we need a special verbosity for such case?

---
In reply to: [81224865](https://github.com/Microsoft/RTVS/pull/2408#discussion_r81224865) [](ancestors = 81224865)